config/config-package.py

```diff
@@ -679,6 +680,7 @@ def configure(self):
                 ".github/workflows/pre-commit.yml",
                 ".gitignore",
                 ".meta.toml",
+                "pyproject.toml",
```

Comment:
As this file might not exist, it should be added to `early_add` in line 629, to make the linter tox run happy.

Reply:
Good catch, I move it.